homeassistant/components/airthings/sensor.py

```diff
@@ -29,40 +31,55 @@
 
 from .const import DOMAIN
 
-SENSORS: dict[str, SensorEntityDescription] = {
-    "radonShortTermAvg": SensorEntityDescription(
+
+@dataclass
+class AirthingsSensorEntityDescription(SensorEntityDescription):
+    """Describes Airthings sensor entity."""
+
+    sensor_type: str | None = None
+
+
+SENSORS: dict[str, AirthingsSensorEntityDescription] = {
+    "radonShortTermAvg": AirthingsSensorEntityDescription(
         key="radonShortTermAvg",
         native_unit_of_measurement="Bq/m³",
+        sensor_type="Radon",
```

Comment:
Why not use `name` attribute?

Reply:
Because I want to include the device name too
`self._attr_name = f"{airthings_device.name} {entity_description.sensor_name}"`

Reply:
`self._attr_name` has priority so there's no conflict.

https://github.com/home-assistant/core/blob/4c40d1767a0d8188e4e6aedfeffd29e68c1939da/homeassistant/helpers/entity.py#L264-L271

Reply:
Ahh, will fix 👍